### PR TITLE
Nikko's Daily Messing Around

### PR DESCRIPTION
**Pull request by Memeji**
# Description

Added a pink collar and coder arm warmers to the vendors and loadout.

---

# Changelog

:cl:
- add: Pink Collar to Clothesmate and Loadout
- add: Coder Arm Warmers to Pride-O-Mat and Loadout

### CODE REVIEW

**Review by ShadesMars — CHANGES_REQUESTED** [review]
There changes that are not marked from the PR, like.

Is changes on LewdTraitSystem supose to be here?

**Comment by Memeji** [comment]
> There changes that are not marked from the PR, like.
> 
> Is changes on LewdTraitSystem supose to be here?

No they are not, those changes are not supposed to be here and will need to be removed. Do I just delete the file from this PR?

**Comment by FoxxoTrystan** [comment]
> > There changes that are not marked from the PR, like.
> > Is changes on LewdTraitSystem supose to be here?
> 
> No they are not, those changes are not supposed to be here and will need to be removed. Do I just delete the file from this PR?

just edit the changes like they are currently on Master, deleting file will **DELETE FILE**

**Comment by Memeji** [comment]
Removed changes to lewdtraitsystem and the lewd components.

**Review by ShadesMars — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Floof/Traits/lewd.yml
[the remark this answers is not in the document]
Comment (Memeji) [reply Resources/Prototypes/Floof/Traits/lewd.yml:1]: Not part of this PR, no.

[the remark this answers is not in the document]
Comment (FoxxoTrystan) [reply Resources/Prototypes/Floof/Traits/lewd.yml:1]: But those changes are here, should that change not be removed?

[the remark this answers is not in the document]
Comment (FoxxoTrystan) [reply Resources/Prototypes/Floof/Traits/lewd.yml:1]: This PR still change the file name/location, no?

**Comment by FoxxoTrystan** [comment]
Stale

**Pull request closed without merging**